### PR TITLE
Remove UGH (8)

### PR DESCRIPTION
**Pull request by matthias-ronge**
Clean up. We would be up to date with the first pull request except for deleting the PICA plugin which dropped out here as discussed.

[compare against part 7](https://github.com/matthias-ronge/kitodo-production/compare/KIT-1155.7...KIT-1155.8)

### CODE REVIEW

File: Kitodo/src/main/java/org/kitodo/production/metadata/copier/MetadataPathSelector.java [hunk 1]
```diff
@@ -217,7 +217,11 @@ protected void createOrOverwrite(CopierData data, LegacyDocStructHelperInterface
      * of the path aren’t available. This works recursively, by calling itself
      * on the subnode, if found, or returning null otherwise.
      *
+<<<<<<< HEAD
```
Comment (Kathrin-Huber) [inline Kitodo/src/main/java/org/kitodo/production/metadata/copier/MetadataPathSelector.java]: you inserted merge-conflict code

File: Kitodo/src/test/resources/log4j2.xml [hunk 2]
```diff
@@ -18,7 +18,7 @@
         </Console>
     </Appenders>
     <Loggers>
-        <Root level="debug">
+        <Root level="error">
```
Comment (Kathrin-Huber) [inline Kitodo/src/test/resources/log4j2.xml]: don't change the log level here

File: Kitodo/src/main/java/org/kitodo/production/metadata/copier/MetadataPathSelector.java (hunk 1 above)
Comment (matthias-ronge) [reply Kitodo/src/main/java/org/kitodo/production/metadata/copier/MetadataPathSelector.java]: Seems to be gone with the latest rebase.

File: Kitodo/src/test/resources/log4j2.xml (hunk 2 above)
Comment (matthias-ronge) [reply Kitodo/src/test/resources/log4j2.xml]: Can’t fix. This is the setting which caused Travis to crash due to too long logs. Obviously the config file was not considered in the past and is considered now (how it shoud be!), don’t ask me why.

Comment (Beacze) [reply Kitodo/src/test/resources/log4j2.xml]: Interesting that until now it was not crashing... It crashes only on this PR. Which stuff is logged here which overload the log? 

_Obviously the config file was not considered in the past and is considered now (how it shoud be!), don’t ask me why._

I don't get what you mean here. It is config file for test logging. Standard logging has own configuration. Even more you, as developer, have in config-local own config file which you can adjust to your needs.

File: Kitodo/src/main/java/org/kitodo/production/metadata/copier/MetadataPathSelector.java (hunk 1 above)
Comment (Beacze) [reply Kitodo/src/main/java/org/kitodo/production/metadata/copier/MetadataPathSelector.java]: Seems not be gone wit the latest rebase - still visible in the file view: https://github.com/kitodo/kitodo-production/blob/ab049a98d126acd2eef0d8587a5bae27173edfb7/Kitodo/src/main/java/org/kitodo/production/metadata/copier/MetadataPathSelector.java

File: Kitodo/src/test/resources/log4j2.xml (hunk 2 above)
Comment (Kathrin-Huber) [reply Kitodo/src/test/resources/log4j2.xml]: I'm going to merge it. I made a Jira Ticket to check log-level

**Pull request merged**